lib/transformations/utils.js

```diff
@@ -83,7 +83,7 @@ function findPluginsRootNodes(j, node) {
 function createProperty(j, key, value) {
 	return j.property(
 		'init',
-		j.literal(key),
+		j.identifier(key),
```

Comment:
This won't work with anything more complex like `foo-bar` but I think it's okay in the scope of this project.